modules/rules/README.md

```diff
@@ -0,0 +1,86 @@
+# SCC Rules module
+
+A module to configure SCC Rules.
+
+Features:
+- Create scc rules for an scc instance
+
+### Usage
+
+```hcl
+module "create_scc_rules" {
+  source          = "../../modules/rules"
```

Comment:
Instead of a relational source for the example, use the official hashicorp module notation which I think would be a source of "terraform-ibm-modules/scc/ibm//modules/rules"? And for the version tag set to "X.X.X".